.github/workflows/docs-check.yml

```diff
@@ -45,5 +45,14 @@ jobs:
       - name: Install mkdocs and dependencies
         run: cd build/release/ && make deps.docs
 
+      - name: Check documentation for CRDs
```

Comment:
Could you (or did you already?) simulate a failure on this PR so we know this check works?

Reply:
Yes it works here is the failure caused due I commit the changes but not didn't commit the CRDs docs just for testing purposes, https://github.com/rook/rook/actions/runs/4826662457/jobs/8598700938?pr=12110

Reply:
I see the docs check is still failing now as well, did you update it so it should pass now?